libs/localization/src/lib/en.json

```diff
@@ -1438,6 +1438,10 @@
         "metricNameColumnHeader": "Metric name",
         "metricDescriptionColumnHeader": "Metric Description"
       },
+      "cohortSelection": {
+        "flyoutHeader": "Choose cohorts",
+        "flyoutDescription": "You can choose to view dataset cohorts or feature-based cohorts. To generate feature-based cohorts you need to first select one or more features in the feature cohorts view."
```

Comment:
"you need to first select"
This text feels incomplete because it doesn't state what needs to be done after selecting the features.  It should be a completed idea by then stating something like "Then, select this <...> button which will enable <...> view."

Reply:
That's fair. How about this:
```
        "flyoutDescription": "You can choose to view dataset cohorts or feature cohorts. If feature cohorts are unavailable you need to first select one or more features in the feature cohorts view. Subsequently, feature cohorts are generated and you can select them here."
```

Reply:
I think that looks much better, it's clearer and doesn't leave the reader thinking about what's next